src/buoyancy.jl

```diff
@@ -6,15 +6,29 @@ const g_Earth = 9.80665
 Supported buoyancy types:
 
 - Nothing 
+- BuoyancyTracer
 - SeawaterBuoyancy
 =#
 
+validate_buoyancy(::Nothing, tracers) = nothing
+
+function validate_buoyancy(buoyancy, tracers) 
+    req_tracers = required_tracers(buoyancy)
+
+    all(tracer ∈ tracers for tracer in req_tracers) || 
```

Comment:
Might be good to stick to either `∈` or `in`. I personally prefer `in` but not too picky.

Reply:
Interesting. Though its true that `in` and `∈` appear to be synonyms --- but via dispatch they have different meanings here; `in` returns a generator whereas `∈` returns a boolean. I appreciate the idea that we should have a smaller vocabulary --- is that what you are suggesting?

Reply:
Ah never mind, that's just me being ignorant. I thought `∈` was the same as `in`, but I guess `∈` returning a boolean is consistent with the mathematical definition.

I guess `∈` more like `is in` but I don't think that `is` is a Julia keyword.